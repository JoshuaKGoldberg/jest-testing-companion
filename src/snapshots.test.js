const result = {
  first: "Spongebob",
  last: "Squarepants",
  profession: "Fry Cook",
  interests: ["Karate", "Jellyfishing"],
};

describe("snapshots", () => {
  test("file", () => {
    expect(result).toMatchSnapshot();
  });

  test("inline", () => {
    expect(result).toMatchInlineSnapshot(`
      Object {
        "first": "Spongebob",
        "interests": Array [
          "Karate",
          "Jellyfishing",
        ],
        "last": "Squarepants",
        "profession": "Fry Cook",
      }
    `);
  });
});
