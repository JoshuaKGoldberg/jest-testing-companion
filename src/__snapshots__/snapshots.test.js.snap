// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file 1`] = `
Object {
  "first": "Spongebob",
  "interests": Array [
    "Karate",
    "Jellyfishing",
  ],
  "last": "Squarepants",
  "profession": "Fry Cook",
}
`;
